.github/CODEOWNERS

```diff
@@ -660,7 +660,7 @@
 # in scripts/linters/pre_commit_linter.py in sync with the modifications.
 # TODO(#11811): Replace @BenHenning with @seanlip after 2021-02-07 for the next
 # 8 lines.
-/core/templates/services/svg-sanitizer.service.ts @BenHenning
+/core/templates/services/img-sanitizer.service.ts @BenHenning
```

Comment:
any specific reason for not writing it `Image-sanitizer.service.ts`?

Reply:
No specific reason. Should I change it to ```image-sanitizer.service.ts```?

Reply:
``image`` is clearer--I suggest using that rather than ``img``.

Reply:
Done